addons/controllers/cpi/vspherecpiconfig_utils.go

```diff
@@ -367,8 +412,8 @@ func (r *VSphereCPIConfigReconciler) getOwnerCluster(ctx context.Context, cpiCon
 	}
 	if err := r.Client.Get(ctx, types.NamespacedName{Namespace: cpiConfig.Namespace, Name: clusterName}, cluster); err != nil {
 		if apierrors.IsNotFound(err) {
-			r.Log.Info(fmt.Sprintf("Cluster resource '%s/%s' not found", cpiConfig.Namespace, clusterName))
-			return nil, nil
+			r.Log.Error(err, fmt.Sprintf("Cluster resource '%s/%s' not found", cpiConfig.Namespace, clusterName))
```

Comment:
no need to add retries here since we now enqueue Cluster's requests

Reply:
I talked to @vijaykatam , since Cluster is required for VSphereCPIConfig Controller to generate the correct config, better to error out otherwise the config wouldn't be correct

Reply:
got it

Reply:
needs this for csi as well